docs/domains/Domain.json

```diff
@@ -243,22 +277,10 @@
           "description": "Configuration for the Administration Server.",
           "$ref": "#/definitions/AdminServer"
         },
-        "useOnlineUpdate": {
-          "description": "Use Online update during lifecycle changes",
-          "type": "boolean"
-        },
-        "rollbackIfRequireStart": {
-          "description": "Rollback dynamic changes if the updates require restart",
-          "type": "boolean"
-        },
         "logHome": {
           "description": "The in-pod name of the directory in which to store the domain, node manager, server logs, and server  *.out files",
```

Comment:
Optional.  The logHome description could be updated to reflect its defaulting behavior.